src/routes/github/webhook/webhook-receiver-post.ts

```diff
@@ -18,14 +19,16 @@ export const WebhookReceiverPost = async (request: Request, response: Response):
 	const payload = request.body;
 	let webhookSecret: string;
```

Comment:
/nit - This is more personal preference than anything for me, but I would split the getWebhookSecret into a seperate function.. Couple reasons.

Its prettier  ¯\_(ツ)_/¯.... maybe, but its much easier for me to read this way
Single responsibility, if we need it again somewhere else(unlikely since its specifically webhook) we can reuse it pretty easy.
This would catch the undefined envVars.WEBHOOK_SECRET, non-existant value on the gitHubServerApp table and gitHubServerApp not found.


```
const getWebhookSecret = async (uuid?: string): string => {
      // Return GitHub cloud secret
      if(!uuid) {
          return envVars.WEBHOOK_SECRET;
      }
      
      // Return GitHub enterprise secret
			const gitHubServerApp = await GitHubServerApp.findForUuid(uuid);
			return gitHubServerApp && await gitHubServerApp.decrypt("webhookSecret");
}
```

```
export const WebhookReceiverPost = async (request: Request, response: Response): Promise<void> => {
....
const webhookSecret = await getWebhookSecret(uuid);

if (!webhookSecret) {
    response.status(400).send("guts");
    return;
}


```